opencti-platform/opencti-graphql/src/modules/case/case.ts

```diff
@@ -31,9 +31,6 @@ const CASE_DEFINITION: ModuleDefinition<StoreEntityCase, StixCase> = {
   attributes: [
     { name: 'name', type: 'string', mandatoryType: 'external', multiple: false, upsert: true },
     { name: 'description', type: 'string', mandatoryType: 'no', multiple: false, upsert: true },
-    { name: 'case_type', type: 'string', mandatoryType: 'internal', multiple: false, upsert: true },
-    { name: 'severity', type: 'string', mandatoryType: 'no', multiple: false, upsert: true },
-    { name: 'priority', type: 'string', mandatoryType: 'no', multiple: false, upsert: true },
     { name: 'x_opencti_workflow_id', type: 'string', mandatoryType: 'no', multiple: false, upsert: true },
     { name: 'rating', type: 'numeric', mandatoryType: 'no', multiple: false, upsert: true },
```

Comment:
I don't see the possibility to add a rating on a Case Incident.
Is it better to drop it in the Feedback class?